SPECS/seabios/seabios.spec

```diff
@@ -17,8 +17,8 @@
 
 Summary:        Open-source legacy BIOS implementation
 Name:           seabios
-Version:        1.14.0
-Release:        7%{?dist}
+Version:        1.16.2
```

Comment:
1.16.3 is also available at https://www.seabios.org/downloads/seabios-1.16.3.tar.gz

Reply:
Updated to the latest version